tests/test_password_reset_submit.py

```diff
@@ -1,19 +1,10 @@
-from pages.pas_reset_page import ResetPassword
-from pages.login_page import LoginPage
+def test_password_reset_submit(login_page_fx, pas_reset_page_fx, valid_user):
+    login_page_fx.open()
+    login_page_fx.forgot_pas_click()
 
+    pas_reset_page_fx.passreset_page_display()
 
-def test_password_reset_submit(browser, valid_user):
-    login_page = LoginPage(browser)
-    forgot_password_page = ResetPassword(browser)
+    pas_reset_page_fx.enter_username(valid_user)
+    pas_reset_page_fx.click_reset_password_button()
 
-    login_page.open()
-    login_page.forgot_pas_click()
-
-    forgot_password_page.passreset_page_display()
-
-    forgot_password_page.enter_username(valid_user)
-    forgot_password_page.click_reset_password_button()
-
-    link_sent_successfully_displayed = forgot_password_page.link_sent_successfully_displayed
-
-    assert link_sent_successfully_displayed, ""
+    assert pas_reset_page_fx.link_sent_successfully_displayed, ""
```

Comment:
It just checks that pas_reset_page_fx object has link_sent_successfully_displayed attribute